test_unstructured/partition/test_api.py

```diff
@@ -202,38 +201,6 @@ def test_partition_via_api_image_block_extraction():
         assert isinstance(image_data, bytes)
 
 
-def test_partition_via_api_pass_list_type_parameters(monkeypatch):
-    mock_request = Mock(return_value=MockResponse(status_code=200))
-    monkeypatch.setattr(requests.Session, "request", mock_request)
```

Comment:
This test was still an integration test despite the patching because it only patched the eventual request made with `requests`. The request is made by the sdk client, though, which has made a number of preprocessing changes before making the request.

in client v0.18.0:
```
def do_request():
    return client.request('POST', url, data=data, files=form, headers=headers)
```

In the current version of the client:
```
def do_request():
    nonlocal req
    try:
        req = client.prepare_request(requests_http.Request('POST', url, params=query_params, data=data, files=form, headers=headers))
        req = self.sdk_configuration.get_hooks().before_request(BeforeRequestContext(hook_ctx), req)
        http_res = client.send(req)
    except Exception as e:
        _, err = self.sdk_configuration.get_hooks().after_error(AfterErrorContext(hook_ctx), None, e)
        if err is not None:
            raise err from e
        raise e
```